drop-boxes/register-omero-metadata/backendinterface.py

```diff
@@ -344,9 +355,13 @@ def add_annotations_to_image(conn, image_id, key_value_data):
 from optparse import OptionParser
 
 ###OMERO server info
-USERNAME = "usr"
-PASSWORD = "pwd"
-HOST = "host"
+#USERNAME = "usr"
+#PASSWORD = "pwd"
+#HOST = "host"
+#PORT = 4064
+USERNAME = "portal_usr_1"
+PASSWORD = "29tesT1ng_4ccez*_04/"
+HOST = "134.2.24.118"
```

Comment:
I hope it is not what i think it is.

Reply:
ahh shot!! my mistake, I will remove this account asap

Reply:
I have changed those credential, they are now invalid